scripts/ansible/oe-windows-acc-setup.yml

```diff
@@ -20,8 +20,8 @@
         seven_zip_hash:      "F00E1588ED54DDF633D8652EB89D0A8F95BD80CCCFC3EED362D81927BEC05AA5"
         vs_buildtools:       "https://oejenkins.blob.core.windows.net/oejenkins/vs_buildtools_2017.exe"
         vs_buildtools_hash:  "6F49872B04A0EAEDF5ED96AB25F7697062D81A419D45D9970E41784F31165BF2"
-        ocaml_url:           "https://oejenkins.blob.core.windows.net/oejenkins/ocpwin64-20160113-4.02.1+ocp1-mingw64.zip"
-        ocaml_hash:          "369F900F7CDA543ABF674520ED6004CC75008E10BEED0D34845E8A42866D0F3A"
+        node_url:            "https://nodejs.org/dist/v10.16.3/node-v10.16.3-x64.msi"
+        node_hash:           "f68b75eea46232adb8fd38126c977dc244166d29e7c6cd2df930b460c38590a9"
```

Comment:
Let's create an issue to cache this node*.msi file in the oejenkins storage account for this ansible playbook